src/OperationProcessor.ts

```diff
@@ -427,11 +467,134 @@ class DidCacheImpl implements DidCache {
       resolvedTransaction,
       opInfo.timestamp.operationIndex);
   }
+
+  private processInternal (opHash: OperationHash, opInfo: OperationInfo): void {
+    // Create operation (which has no parents) is handled differently from the other
+    // operations which do have parents.
+    if (opInfo.type === OperationType.Create) {
+      this.processCreateOperation(opHash, opInfo);
+    } else {
+      this.processOperationWithParent(opHash, opInfo);
+    }
+
+    // Process operations waiting on this operation
+    this.processDescendantsWaitingOn(opHash);
+  }
+
+  private processCreateOperation (_opHash: OperationHash, opInfo: OperationInfo): void {
+    // TODO: Validate create operation (verify signature)
+    opInfo.status = OperationStatus.Valid;
+  }
+
+  private processOperationWithParent (opHash: OperationHash, opInfo: OperationInfo): void {
+    const parentOpHash = opInfo.parent as OperationHash;
```

Comment:
Consider using `opInfo.parent!` instead, same goes for all occurrences.

Reply:
I did not know about this operator. Changed the code in a lot of places to use this operator.